# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/helloworld.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-5.18/arch/powerpc/platforms/powermac/udbg_scc.c.vanilla	2022-05-29 13:43:25.840760694 +0200
+++ linux-5.18/arch/powerpc/platforms/powermac/udbg_scc.c	2022-05-29 13:44:09.047758971 +0200
@@ -148,7 +148,7 @@
 	udbg_getc = udbg_scc_getc;
 	udbg_getc_poll = udbg_scc_getc_poll;
 
-	udbg_puts("Hello World !\n");
+	udbg_puts("udbg_scc_init\n");
 
  bail:
 	of_node_put(macio);
